layouts/partials/header.html

```diff
@@ -31,11 +31,11 @@
     <meta property="og:type" content="website">
     {{ $featured := (.Resources.ByType "image").GetMatch "*featured*" }}
     {{ with $featured }}
-        <meta property="og:image" content="{{ $featured.RelPermalink }}">
+        <meta property="og:image" content="https://camel.apache.org{{ $featured.RelPermalink }}">
```

Comment:
@zregvart I finally made some test and your solution looks good. Do you know if there is some property on the `$featured` image to directly get the full url by any chance?